technical-reference/reference-technical-articles/technical-doc-xvs-bridge.md

```diff
@@ -0,0 +1,174 @@
+
+<figure><img src="../../.gitbook/assets/bnb-dest1.jpg" alt="Assets bridging from src chain to dest chain"><figcaption></figcaption></figure>
+
+1. Call the `sendFrom` function of the Bridge contract with the following parameters:
+   - `_from`: Your address on the BNB chain
+   - `_dstChainId`: Destination Chain ID [defined by LayerZero](https://layerzero.gitbook.io/docs/technical-reference/mainnet/supported-chain-ids) (e.g., Ethereum virtual chain ID (101))
+   - `_toAddress`: The address on the Destination Chain where you want to receive the XVS tokens
+   - `_amount`: Amount of XVS tokens you want to send, defined with 18 decimals
+   - `_callParams`: ["RefundGasAddress", "ZROaddress", "adapterParams"]
+     - `RefundGasAddress`: Address where you want to receive a refund for excessive gas sent. It can be the sender's address.
+     - `ZROaddress`: `0x0000000000000000000000000000000000000000` (indicating that you are not paying in ZRO tokens)
+     - `adapterParams`: `0x000100000000000000000000000000000000000000000000000000000000000493E0` (ethers.utils.solidityPack(['uint16','uint256'],[1, gasValue]) the gas value should be greater then minDestGas which is set to 300k).
+
+## 3. Receiving Tokens on the Destination Chain
+
+When you send XVS tokens to the Destination Chain using the Bridge, the tokens will be minted by the Bridge contract to the receiver's address on the Destination Chain.
+
+## 4. Transferring Tokens Back to the BNB chain
+
+<figure><img src="../../.gitbook/assets/dest-bnb.jpg" alt="Assets bridging from dest chain to src chain"><figcaption></figcaption></figure>
+
+To transfer XVS tokens back to the BNB chain, follow a similar process as mentioned in the earlier send section. You don't need to approve the Bridge contract on the Destination Chain to spend XVS tokens on your behalf. The tokens will be burned on the Destination Chain on your behalf and unlocked and transferred to the receiver's address on the BNB chain.
+
+## 5. Monitoring Transaction Status
+
+After initiating a token transfer, you should wait for the transaction to confirm. This process may take a few minutes. Once the transaction confirms, you will receive the bridged XVS tokens on the Destination Chain. You can use [LayerZero scan](https://layerzeroscan.com) to monitor your cross-chain transactions.
+
+## 6. Security and Risks
+
+<figure><img src="../../.gitbook/assets/risk.jpg" alt="Risks and security"><figcaption></figcaption></figure>
+
+### 6.1. Ownership Transfer
+
+- Use the `transferOwnership` method in the XVSBridgeAdmin contract to transfer ownership of the admin contract.
+- Use the `transferBridgeOwnership` method to transfer ownership of the Bridge contract from one contract to another.
+- Ownership control is crucial in case of emergencies or security issues.
+- The owner of the XVSBridgeAdmin contract will be initially the Guardian, but it will be transferred to Governance as soon as the the Multichain Governance module is deployed.
```

Comment:
```suggestion
- The owner of the `XVSBridgeAdmin` contract will be initially the `Guardian`, but it will be transferred to Governance as soon as the `MultichainGovernance` module is deployed.
```

Is MultichainGovernance a contract?